src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/preview.html.twig

```diff
@@ -0,0 +1,121 @@
+            {{ 'Invoice details'|trans({}, 'Admin.Global') }}:
+          </th>
+          <th class="mb-1">
+            <i class="material-icons">local_shipping</i>
+            {{ 'Shipping details'|trans({}, 'Admin.Global') }}:
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>{{ orderPreview.invoiceDetails.firstName }} {{ orderPreview.invoiceDetails.lastName }}</td>
+          <td>{{ orderPreview.invoiceDetails.firstName }} {{ orderPreview.shippingDetails.lastName }}</td>
+        </tr>
+
+        <tr>
+          <td>{{ orderPreview.invoiceDetails.company }}</td>
+          <td>{{ orderPreview.shippingDetails.address1 }}</td>
+        </tr>
+
+        <tr>
+          <td>{{ orderPreview.invoiceDetails.address1 }}</td>
+          <td>{{ orderPreview.shippingDetails.address2 }}</td>
+        </tr>
+
+        <tr>
+          <td>{{ orderPreview.invoiceDetails.address2 }}</td>
+          <td>{{ orderPreview.shippingDetails.country }}</td>
+        </tr>
+
+        <tr>
+          <td>{{ orderPreview.invoiceDetails.country }}</td>
+          <td>{{ orderPreview.shippingDetails.phone }}</td>
+        </tr>
+
+        <tr>
+          <td>{{ orderPreview.invoiceDetails.phone }}</td>
+        </tr>
+
+        <tr>
+          <td>{{ orderPreview.invoiceDetails.email }}</td>
+        </tr>
+      </tbody>
+    </table>
+    <div class="offset-2 col-6 ">
+      <div class="table-responsive">
+        <table class="table product table-borderless border-bottom-0">
+          <thead>
+          <tr>
+            {% if orderPreview.taxIncluded %}
+              {% set taxInclusion = 'Tax included'|trans({}, 'Admin.Catalog.Feature') %}
+            {% else %}
+              {% set taxInclusion = 'Tax excluded'|trans({}, 'Admin.Catalog.Feature') %}
+            {% endif %}
+
+            <th>{{ 'Product'|trans({}, 'Admin.Global') }}</th>
+            <th>
+              {{ 'Base price'|trans({}, 'Admin.Global') }}
+              <small>{{ taxInclusion }}</small>
+            </th>
+            <th>{{ 'Quantity'|trans({}, 'Admin.Global') }}</th>
+            <th>
+              {{ 'Total'|trans({}, 'Admin.Global') }}
+              <small>{{ taxInclusion }}</small>
+            </th>
+          </tr>
+          </thead>
+          <tbody>
+          {% for productDetail in orderPreview.productDetails %}
+            <tr>
+              <td class="p-1">{{ productDetail.name }}</td>
+              <td class="p-1">{{ productDetail.unitPrice }}</td>
+              <td class="p-1">{{ productDetail.quantity }}</td>
+              <td class="p-1">{{ productDetail.totalPrice }}</td>
+            </tr>
+          {% endfor %}
+          </tbody>
+        </table>
+      </div>
+
+      <div class="text-right">
+        <a href="{{ path('admin_orders_view', {'orderId': orderId}) }}" class="btn btn-primary btn-sm">
+          {{ 'Open order'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
Open order? Does it mean 'see order'?

Reply:
Yeah, it opens order view page. I followed new UI design which had this wording.

Reply:
Alright, it looks weird since it is not a wording often employed in PrestaShop and, also, it does not open a popup but the order page. Here are my suggestions for this button:

_View order_ in _Admin.Orderscustomers.Feature_
_View full details_ in _Admin.Actions_
_See more_ in _Admin.Actions_

cc @MatShir

Reply:
So which wording should I use @LouiseBonnard ? :)

Reply:
I have even better! I suggest _Order details_ in _Admin.Orderscustomers.Feature_, it will be consistent with the order wordings on this page.
💡

Reply:
Done!